### PR TITLE
Athena neptune bugfix - Null value handling and Type casting 

### PR DESCRIPTION
**Pull request by sandeepveldi**
*Issue #, if available:*

*Description of changes:*
Fixed the null value handling and type casting issues in the Row Writer.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/TypeRowWriter.java [hunk 1]
```diff
@@ -38,18 +38,19 @@
 
 import java.util.ArrayList;
 import java.util.Map;
+
 /**
  * This class is a Utility class to create Extractors for each field type as per
  * Schema
  */
-public final class TypeRowWriter
+public final class TypeRowWriter
```
Comment (avirtuos) [inline athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/TypeRowWriter.java:46]: i think your IDE is using a different style than we are. can you remove these extra whitespace changes? there are many of them

File: athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java [hunk 2]
```diff
@@ -167,7 +167,7 @@ private void buildGraphTraversal() {
                 Vertex vertex1 = tinkerGraph.addVertex(T.label, "default");
                 vertex1.property("property1", 10);
                 vertex1.property("property2", "string1");
-                vertex1.property("property3", 12.4);
+                vertex1.property("property3", 12);
```
Comment (avirtuos) [inline athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java:170]: why this change? can you add another property instead of changing existing?

File: athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java [hunk 3]
```diff
@@ -181,7 +181,6 @@ private void buildGraphTraversal() {
                 Vertex vertex3 = tinkerGraph.addVertex(T.label, "default");
                 vertex3.property("property1", 9);
                 vertex3.property("property2", "string3");
-                vertex3.property("property3", 15.4);
                 vertex3.property("property4", true);
```
Comment (avirtuos) [inline athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java:184]: why removing?

**Review by avirtuos — CHANGES_REQUESTED** [review]
looks pretty good but the test case changes were unclear to me. can you give an example of what caused the issue before?

File: athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java (hunk 3 above)
Comment (sandeepveldi) [reply athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java:184]: We want to add test case to test the null value handling when a particular field value is null/empty

File: athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/TypeRowWriter.java (hunk 1 above)
Comment (sandeepveldi) [reply athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/TypeRowWriter.java:46]: Will update the code by removing white spaces.

File: athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java (hunk 2 above)
Comment (sandeepveldi) [reply athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java:170]: Also, the bugs that we found are:

1. Failure to detect null values returned from the data store and handling those gracefully before returning the result set to Athena.
2. Type casting issues for certain data types such as Double.

Comment (avirtuos) [reply athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java:170]: i'm not sure this answers my question of why you removed the 12.4 test case as opposed to adding a new one.

File: athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java (hunk 3 above)
Comment (avirtuos) [reply athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java:184]: you removed a test case here, can you add a new property to test the null/empty case?

**Review by avirtuos — CHANGES_REQUESTED** [review]

File: athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java (hunk 2 above)
Comment (abhishekpradeepmishra) [reply athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java]: added new test cases

File: athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java (hunk 3 above)
Comment (abhishekpradeepmishra) [reply athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java]: added new test cases

**Review by avirtuos — APPROVED** [review]
